cmd/egress/cmd.go

```diff
@@ -199,6 +202,7 @@ are set correctly before execution.
 		},
 	}
 
+	validateEgressCmd.Flags().StringVar(&config.platformType, "platform", platformTypeDefault, fmt.Sprintf("(optional) infra platform type, which determines which endpoints to test. Either '%[1]v', '%[2]v', or '%[3]v' (hypershift)", helpers.PLATFORM_AWS, helpers.PLATFORM_GCP, helpers.PLATFORM_HOSTEDCLUSTER))
```

Comment:
I don't think we need the explicit indexing here, especially since we're not doing any repeating/out-of-order prints
```suggestion
	validateEgressCmd.Flags().StringVar(&config.platformType, "platform", platformTypeDefault, fmt.Sprintf("(optional) infra platform type, which determines which endpoints to test. Either '%v', '%v', or '%v' (hypershift)", helpers.PLATFORM_AWS, helpers.PLATFORM_GCP, helpers.PLATFORM_HOSTEDCLUSTER))
```

Reply:
Agreed, and addressed in the latest commit :)